backend/test/integration/upgrade_test.go

```diff
@@ -78,12 +78,15 @@ func (s *UpgradeTests) SetupSuite() {
 		return
 	}
 
-	err := test.WaitForReady(*namespace, *initializeTimeout)
-	if err != nil {
-		glog.Exitf("Failed to initialize test. Error: %v", err)
+	if !*isDevMode {
+		err := test.WaitForReady(*namespace, *initializeTimeout)
+		if err != nil {
+			glog.Exitf("Failed to initialize test. Error: %v", err)
+		}
 	}
 	s.namespace = *namespace
 	clientConfig := test.GetClientConfig(*namespace)
+	var err error
 	s.experimentClient, err = api_server.NewExperimentClient(clientConfig, false)
```

Comment:
nit: I think you can use `s.experimentClient, err :=`, no? Then you don't have to define `var err error`. 
Or if you have to define it, maybe it's better to have it on line 80 (with change on line 82)

Reply:
I cannot use `:=` because, the first var is `s.experimentClient`, it cannot be declared. (I tried that first.)

Thanks, moving it to line 80 sounds reasonable.

Reply:
updated, PTAL